CSharp/VirtoCommerce.MemberExModule.Template/Data/Migrations/201906110802132_Initial.cs

```diff
@@ -39,6 +36,7 @@ public override void Down()
             DropForeignKey("dbo.MemberSupplier", "Id", "dbo.Member");
             DropForeignKey("dbo.MemberSupplierReview", "SupplierId", "dbo.MemberSupplier");
 
```

Comment:
remove empty lines

Reply:
Extra empty line removed. The rest too to delete? (Other templates also use spaces in such situations)

Reply:
also remove empty line 32. And yes, remove empty lines in all templates

Reply:
Good, deleted